### PR TITLE
Patch to allow HTML link processing

### PR DESCRIPTION
**Pull request by ChadEmory**
Please import the following patch to enable HTML links to be processed by the Script.

### CODE REVIEW

File: SheetConverter/Code.gs [hunk 1]
```diff
@@ -291,11 +308,21 @@ function convertRange2html(range){
           break;
         }
       }
+      
       if (!isTdPartOfRange) {
+        Logger.log("row - " + row + " col - " + col);
```
Comment (mogsdad) [inline SheetConverter/Code.gs:313]: Remove `Logger` calls.

Comment (mogsdad) [inline SheetConverter/Code.gs:313]: Remove `Logger` calls.

**Review by mogsdad — CHANGES_REQUESTED** [review]

**Comment by ChadEmory** [comment]
Removed all logger references in my version of the code. I also noticed there is more Logger entries in other areas of the code that you might want to clear out as well.

**Comment by mogsdad** [comment]
> Removed all logger references in my version of the code. I also noticed there is more Logger entries in other areas of the code that you might want to clear out as well.

There is only one instance in code (vs in comment examples), and it serves the specific purpose of reporting back to the user that the format they have provided is not supported. That's a stark difference from leftover debugging code.

**Comment by mogsdad** [comment]
Thanks for the PR, Chad - merged.

**Pull request merged**